www/gatsby-node.js

```diff
@@ -443,16 +446,29 @@ exports.createPages = ({ graphql, actions, reporter }) => {
             )
           }
 
-          createPage({
-            path: `${node.fields.slug}`, // required
-            component: slash(
-              node.fields.package ? localPackageTemplate : docsTemplate
-            ),
-            context: {
-              slug: node.fields.slug,
-              ...nextAndPrev,
-            },
-          })
+          if (node.frontmatter.jsdoc) {
+            createPage({
+              path: `${node.fields.slug}`, // required
+              component: slash(apiTemplate),
+              context: {
+                slug: node.fields.slug,
+                jsdoc: node.frontmatter.jsdoc,
+                apicalls: node.frontmatter.apicalls,
+                ...nextAndPrev,
+              },
+            })
+          } else {
+            createPage({
+              path: `${node.fields.slug}`, // required
+              component: slash(
+                node.fields.package ? localPackageTemplate : docsTemplate
```

Comment:
Maybe split this out into three things so it's more obvious what the choices are? i.e.

```
if (node.frontmatter.jsdoc) {
 // render apiTemplate
} else if (node.fields.package) {
 // render localPackageTemplate
} else {
 // render docsTemplate
}